### ISSUE TITLE
Question: plans for 1.0?

### ISSUE THREAD

**Issue by trevordmiller** [apollographql/apollo-codegen#237]
I'd like to use `apollo-codegen` to automate generating Flow types from our schema types. I'm currently using [gql2flow](https://github.com/joarwilk/gql2flow) which works well but has two problems:
1) Everything is nullable to allow for composing fields - so a lot of unnecessary null checks are needed; it seems apollo-codegen fixes this by requiring `.graphql` files per component which generate per-query types.
2) The file output from the library is non-deterministic; the diff is different almost every time it is run which needlessly complicates code reviews.

However, looking over apollo-codegen, it seems it introduces more problems around custom scalars , nested object fields, and [a bunch of other open issues](https://github.com/apollographql/apollo-codegen/issues). So, it seems this library is experiencing a decent amount of churn.

Bottom line: are there plans for a 1.0 release where this library will be a bit more stable for use with production apps? Thank you!

**Comment by martijnwalraven** [comment]
Yep, we're very interested in getting to a 1.0 release soon. There have been some pretty major changes to the compiler that should help fix some of these issues, but we still need to update the TypeScript/Flow targets to take advantage of that. If you're interested in helping out, you may want to join the `#apollo-codegen` channel on the [Apollo Slack](http://www.apollographql.com/#slack).
